single.php

```diff
@@ -17,7 +17,10 @@
 
 			get_template_part( 'template-parts/content', get_post_type() );
 
-			the_post_navigation();
+			the_post_navigation( array(
+				'prev_text' => '<span class="nav-subtitle">' . __( 'Previous:', '_s' ) . '</span> <span class="nav-title">%title</span>',
```

Comment:
The call to `__()` should be replaced with `esc_html__()`, both in this line and the line below.